<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SPARQLWidget</class>
 <widget class="QWidget" name="SPARQLWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SPARQL</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>SPARQL Queries</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPlainTextEdit" name="queryEdit">
          <property name="plainText">
           <string>SELECT * WHERE { ?a ?b ?c . }</string>
          </property>
          <property name="placeholderText">
           <string notr="true">SELECT ?s ?p ?o WHERE { ?s ?p ?o . }</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>442</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnQuery">
            <property name="text">
             <string>Submit</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeView" name="queryResultView"/>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QListView" name="queryList"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
